src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs

```diff
@@ -304,7 +304,7 @@ private protected override void SaveModel(ModelSaveContext ctx)
             ctx.CheckAtModel();
             ctx.SetVersionInfo(GetVersionInfo());
 
-            ctx.SaveBinaryStream("OnnxModel", w => { w.WriteByteArray(File.ReadAllBytes(Model.ModelFile)); });
+            ctx.SaveBinaryStream("OnnxModel", w => { w.WriteByteArray(File.ReadAllBytes(Model.ModelStream.Name)); });
```

Comment:
Is this change intentional?

Reply:
Yeah, Before we just tracked the file name, but if the user didn't dispose of the OnnxTransformer the file would stay on disk. Now, we are keeping a file handle with `Delete on close` set, so this change is just to get the name from the handle instead.